ambry-quota/src/main/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresher.java

```diff
@@ -150,8 +150,6 @@ private void initializeContainerStorageUsageMonthlyBase() {
           }
         } catch (Exception e) {
           throw new IllegalStateException("Unable to fetch monthly storage usage from mysql", e);
-        } finally {
-          accountStatsStore.closeConnection();
```

Comment:
Since we are using connection pool now, just thinking if we still need synchronization on access to `accountStatsStore` in few of the methods in this class? Seems like it is removed on other classes like `StatsManager` and `MySqlReportAggregatorTask`.

Reply:
updated, forgot to remove the synchroinzed block in this file.